packages/amplication-server/src/core/moduleAction/dto/EnumModuleActionGqlOperation.ts

```diff
@@ -0,0 +1,10 @@
+import { registerEnumType } from "@nestjs/graphql";
+
+export enum EnumModuleActionGqlOperation {
```

Comment:
dont you have this enum on code gen type ?

Reply:
@barshimi No...